src/views/PopularMovieView.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <CardMovie :title="titleForMoviePage" :movies="popularMovies" />
+</template>
+
+<script>
```

Comment:
I think you already have Popular movies component. It could be deleted

Reply:
Deleted